### PR TITLE
Update build tools and target SDK version to support evernote job scheduling

### PR DESCRIPTION
**Pull request by deanrobertcook**
## @QA
This should fix the case of missing messages we've seen when turning off airplane mode. It's not perfectly correct yet, but we'll introduce further fixes later

Question to self - what's our procedure for using third party libraries and providing recognition - will fix before merging 
#### APK
[Download build #9693](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9693/artifact/build/artifact/wire-dev-PR1219-9693.apk)
[Download build #9696](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9696/artifact/build/artifact/wire-dev-PR1219-9696.apk)
[Download build #9705](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9705/artifact/build/artifact/wire-dev-PR1219-9705.apk)
[Download build #9706](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9706/artifact/build/artifact/wire-dev-PR1219-9706.apk)

### CODE REVIEW

File: app/src/main/java/com/waz/zclient/utils/DebugUtils.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public static String getVersion(Context context) {
         versionText.append("\nSync Engine ").append(ZmsVersion.ZMS_VERSION)
                    .append("\nAVS ").append(context.getString(R.string.avs_version))
                    .append("\nAudio-notifications ").append(context.getString(R.string.audio_notifications_version))
-                   .append("\nTranslations ").append(context.getString(R.string.wiretranslations_version))
+//                   .append("\nTranslations ").append(context.getString(R.string.wiretranslations_version))
```
Comment (CalumMcCall) [inline app/src/main/java/com/waz/zclient/utils/DebugUtils.java]: Why comment this line out? Would it not be better to just delete it?

File: app/src/main/scala/com/waz/zclient/controllers/AssetsController.scala [hunk 2]
```diff
@@ -181,8 +181,8 @@ class AssetsController(implicit context: Context, inj: Injector, ec: EventContex
     title.setGravity(Gravity.CENTER)
 
     val openButton = dialog.findViewById(R.id.ttv__file_action_dialog__open).asInstanceOf[TextView]
-    val noAppFoundLabel = dialog.findViewById(R.id.ttv__file_action_dialog__open__no_app_found)
-    val saveButton = dialog.findViewById(R.id.ttv__file_action_dialog__save)
+    val noAppFoundLabel = dialog.findViewById(R.id.ttv__file_action_dialog__open__no_app_found).asInstanceOf[View]
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/controllers/AssetsController.scala]: Are these calls not redundant?

**Comment by deanrobertcook** [comment]
build again

**Comment by deanrobertcook** [comment]
build again

**Comment by deanrobertcook** [comment]
build again

**Comment by someoneEsle** [comment]
Have you tested this? Because this is what the first screen looks like.
wire 2017-10-25 at 17 04 41

**Comment by deanrobertcook** [comment]
let's try again :)

**Comment by someoneEsle** [comment]
OK, let's try merging this into internal and monitor. I'll keep an eye on it over the next few days

**Pull request merged**